tests/unit/test_cromshellconfig.py

```diff
@@ -0,0 +1,97 @@
+from cromshell.utilities import cromshellconfig
+from importlib import reload
+from pathlib import Path
+import pytest
+import os
+
+
+class TestCromshellConfig:
+    """Test the cromshell config functions and variables"""
+
+    def test_override_slim_metadata_parameters(self):
+        reload(cromshellconfig)
+        default_slim_parameters = (
+            "=includeKey=id&includeKey=executionStatus&includeKey=backendStatus"
+            "&includeKey=status&includeKey=callRoot&expandSubWorkflows=true&includeKey"
+            "=subWorkflowMetadata&includeKey=subWorkflowId"
+        )
+        assert cromshellconfig.slim_metadata_parameters == default_slim_parameters, \
```

Comment:
Therefore this test may be removed.
In OOP, we may test that the constructors correctly initialize data in instances. In this case there's no ctors involved, so we could forgo that.